core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -50,6 +50,14 @@ public void register(BuiltinFunctionRepository repository) {
     repository.register(match_phrase(BuiltinFunctionName.MATCH_PHRASE));
     repository.register(match_phrase(BuiltinFunctionName.MATCHPHRASE));
     repository.register(match_phrase_prefix());
+    repository.register(highlight());
+  }
+
+  private static FunctionResolver highlight() {
+    FunctionName functionName = BuiltinFunctionName.HIGHLIGHT.getName();
+    FunctionSignature functionSignature = new FunctionSignature(functionName, List.of(STRING));
+    FunctionBuilder functionBuilder = arguments -> new HighlightExpression(arguments.get(0));
```

Comment:
can we assume arguments always has one item?  otherwise we could get a nullpointerexception

Reply:
The parser will fail with no arguments, or more than one argument.